### PR TITLE
Add support build no cache

### PR DESCRIPTION
**Pull request by bohai**
Add support build --no-cache

Signed-off-by: Bo Hai <bohai@huawei.com>

### CODE REVIEW

**Comment by bohai** [comment]
ping @ibuildthecloud

File: integration/basic_test.go [hunk 1]
```diff
@@ -264,6 +264,29 @@ func (s *RunSuite) TestBuild(c *C) {
 	c.Assert(two.Config.Cmd, DeepEquals, []string{"echo", "two"})
 }
 
+func (s *RunSuite) TestBuildWithNoCache(c *C) {
```
Comment (vdemeester) [inline integration/basic_test.go]: I think the integration test(s) should do more than that ; something like :
- A test for cached build :
  - `build` the image (without the `--no-cache`)
  - `build` the image again (without the `--no-cache`) and make sure we see `Using cache` somewhere
- A test for the `no-cache` build :
  - `build` the image (without the `--no-cache`)
  - `build` the image with `--no-cache` and make sure we don't see `Using cache` messages.
 
The actual test that the `build` work and did not fail with the `--no-cache` flag, but does not check at all if it used the cache or not :stuck_out_tongue_closed_eyes:.

**Comment by vdemeester** [comment]
/ping @bohai could you take care of my comments ?
PS : I can carry it and update the integration-test in an additional commit if you can't :stuck_out_tongue_closed_eyes:.

**Comment by bohai** [comment]
@vdemeester 
Thanks for your comment. 
I will try to add integration-test as soon as possible.

**Comment by bohai** [comment]
I feel there is something wrong with the output of  build command.

This is the outputs i got:
<pre><code>
root@SZX1000041895:~/build# libcompose build
WARN[0000] Note: This is an experimental alternate implementation of the Compose CLI (https://github.com/docker/compose)
INFO[0000] Building build_one...
Successfully built 2a823592c804
INFO[0000] Building build_two...
Step 1 : FROM busybox
 ---> 2c5ac3f849df
Step 2 : CMD echo two
 ---> Using cache
 ---> 1db7a126d491
Successfully built 1db7a126d491
root@SZX1000041895:~/build# libcompose build
WARN[0000] Note: This is an experimental alternate implementation of the Compose CLI (https://github.com/docker/compose)
INFO[0000] Building build_one...
Successfully built 2a823592c804
INFO[0000] Building build_two...
Step 1 : FROM busybox
Successfully built 1db7a126d491
root@SZX1000041895:~/build# libcompose build
WARN[0000] Note: This is an experimental alternate implementation of the Compose CLI (https://github.com/docker/compose)
INFO[0000] Building build_one...
Step 1 : FROM busybox
 ---> 2c5ac3f849df
Step 2 : CMD echo one
 ---> Using cache
 ---> 2a823592c804
Successfully built 2a823592c804
INFO[0000] Building build_two...
Step 1 : FROM busybox
 ---> 2c5ac3f849df
Step 2 : CMD echo two
 ---> Using cache
 ---> 1db7a126d491
Successfully built 1db7a126d491
</code></pre>

**Comment by vdemeester** [comment]
@bohai definitly something weird on `build` :sob:

**Comment by vdemeester** [comment]
@bohai #120 merged, I'm LGTM on this one so I'll carry it :wink:

**Pull request closed without merging**